.github/workflows/okhi_ci_cd.yml

```
@@ -0,0 +1,126 @@
+permissions:
+  contents: write   # allow creating/updating Releases & uploading assets
+  actions:  read
+
+jobs:
+  build-ios:
+    name: Build iOS for TestFlight
+    runs-on: macos-latest
+    needs: build-android
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)